wolfcrypt/src/wc_debug.c

```diff
@@ -24,14 +24,37 @@
     #include <config.h>
 #endif
 
+#include <wolfssl/options.h>
```

Comment:
Please add the `#ifndef WOLFSSL_USER_SETTINGS` logic here. Otherwise embedded systems building without configure will be missing options.h.